tools/dev/update_wordpress_vulnerabilities.rb

```diff
@@ -74,7 +74,7 @@ def cleanup_text(txt)
 Dir.glob(path + '/modules/**/*.rb').each do |file|
   next unless file.include?('exploits') || file.include?('auxiliary')
 
-  str = IO.read(file)
+  str = File.binread(file)
```

Comment:
Let's replace this with `File.read` for explicitness. We don't want to `binread` here, as we would want to continue converting parsing `\r\n` correctly on windows boxes

Reply:
Looks like this has already been resolved in the branch with the proposed solution of `File.read`, so going to mark this as a resolved issue.